### PR TITLE
Add license/repo information to published jars/poms

### LINKED ISSUE #14: Missing information in published files
The publish plugin jars/poms do not contain information like LICENSE/NOTICE, homepage, scmInfo, etc.

### PR DESCRIPTION
**Pull request by stevedlawrence**
- Add LICENSE/NOTICE to META-INF directory so they are includd in the published jars
- Add meta information about homepage, scm, etc to build.sbt so it is included in the pom

Closes #14

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -27,6 +27,19 @@ scalacOptions ++= Seq(
   "-Ywarn-unused:imports",
 )
 
+scmInfo := Some(
```
Comment (mbeckerle) [inline build.sbt:30]: I completely misunderstood the intent of this PR. 

I thought it was about making sure that a project that uses sbt-daffodil could have its LICENSE/NOTICES etc. included in its jars, and appropriate metadata into poms. 

But this is just about this sbt plugin project itself.

**Review by mbeckerle — APPROVED** [review]
+1

File: build.sbt (hunk 1 above)
Comment (stevedlawrence) [reply build.sbt:30]: Correct, this is just about making Apache happy when we eventually create a release of this plugin. I'll modify the commit message to make that more obvious before pushing.

Note that we do have issue https://github.com/apache/daffodil-sbt/issues/7 about copying REAMDEs into schema project jars. I'll update that to also mentions copying in LICENSE information.

**Review by tuxji — APPROVED** [review]
+1

**Pull request merged**